TestBenches/VMRouterCM_test.cpp

```diff
@@ -21,7 +21,7 @@
 
 using namespace std;
 
-const int nEvents = 100;  //number of events to run
+const int nEvents = 3;  //number of events to run
```

Comment:
@aryd Please keep the default number of events to test at 100 for the CI.
```suggestion
const int nEvents = 100;  //number of events to run
```